mungegithub/build-cop/emergency.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+STOP=${1:-"stop"}
```

Comment:
"resume"-- there's a message that prints it below.

On Fri, Jun 17, 2016 at 10:53 AM, Mike Danese <notifications@github.com>
wrote:

> In mungegithub/build-cop/emergency.sh
> <https://github.com/kubernetes/contrib/pull/1228#discussion_r67549989>:
>
> > +# you may not use this file except in compliance with the License.
> > +# You may obtain a copy of the License at
> > +#
> > +#     http://www.apache.org/licenses/LICENSE-2.0
> > +#
> > +# Unless required by applicable law or agreed to in writing, software
> > +# distributed under the License is distributed on an "AS IS" BASIS,
> > +# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +# See the License for the specific language governing permissions and
> > +# limitations under the License.
> > +
> > +set -o errexit
> > +set -o nounset
> > +set -o pipefail
> > +
> > +STOP=${1:-"stop"}
>
> what are the other options?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1228/files/00884b828ba73b604c62bd1e7805f2e82cb6cee5#r67549989>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAnglo5351ZsoJVhf5qed70bs7L_go_rks5qMt8EgaJpZM4I4k-d>
> .
>